app/sidekiq/evss/disability_compensation_form/submit_form526_cleanup.rb

```diff
@@ -6,12 +6,53 @@ module EVSS
   module DisabilityCompensationForm
     class SubmitForm526Cleanup < Job
       include Sidekiq::Job
-      STATSD_KEY_PREFIX = 'worker.evss.submit_form526_cleanup'
+      STATSD_KEY = 'worker.evss.submit_form526_cleanup.exhausted'
 
       # This callback cannot be tested due to the limitations of `Sidekiq::Testing.fake!`
       # :nocov:
```

Comment:
double check spec is running without this (remove it)

Reply:
Confirmed. Removed.

The spec will evaluate all included tests regardless of whether SimpleCov's code coverage analysis is disabled for a particular method by wrapping code in `:nocov:` comments. In this particular case, code coverage is 100% if the `sidekiq_retries_exhausted` block is not analyzed, whereas 92.31% of the lines are covered when reenabling coverage. Reference following screenshots. 

![coverage disabled](https://github.com/department-of-veterans-affairs/vets-api/assets/50811220/8ba15afa-334c-419b-abd8-0154e3c85f26)
<img width="505" alt="coverage enabled" src="https://github.com/department-of-veterans-affairs/vets-api/assets/50811220/08d1b9cd-a626-488b-95c6-883841685bf7">